### PR TITLE
Adding troubleshooting steps for unknown glyps icon in the prompt.

### PR DESCRIPTION
**Pull request by MrMeemus**
Due to the inconsistency of reproducing #111, I am providing a trouble shooting section on the usage guide.

### CODE REVIEW

File: doc/troubleshooting_guide.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+## <a name="Usage_Issues"></a>Usage Issues:
+
+### Unknown glyph fills up prompt:
+If you encounter the display below, it is a Windows 10 issue that can pop up on the command prompt or powershell prompt.
+The current workaround for this issue is to change the font of the prompt to Consolas.
+![alt text](https://github.com/dbcli/mssql-cli/tree/master/screenshots/mssql-cli-display-issue.png "mssql-cli display issue")
+
+
+### <a name="Reporting_Issues"></a>Reporting Issues:
```
Comment (pensivebrian) [inline doc/troubleshooting_guide.md]: Can we remove the anchor tags...they aren't needed

File: doc/troubleshooting_guide.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+If the issue you are encountering is not listed above nor filed on our github, please file a issue here [issues](#https://github.com/dbcli/mssql-cli/issues) with the following information listed below and any other additional symptoms of your issue.
```
Comment (pensivebrian) [inline doc/troubleshooting_guide.md]: Let's add 'which pip' and 'which python' to the commands to report.

Comment (pensivebrian) [reply doc/troubleshooting_guide.md]: Can we also have the user post the commands used to install mssql-cli?

Comment (MrMeemus) [reply doc/troubleshooting_guide.md]: Done.

File: doc/troubleshooting_guide.md (hunk 1 above)
Comment (MrMeemus) [reply doc/troubleshooting_guide.md]: I think we need the anchors to reference our headers since they have spaces in between them.

**Pull request merged**